anchor/programs/pubkey-protocol/src/instructions/community/verify_profile.rs

```diff
@@ -0,0 +1,72 @@
+use anchor_lang::prelude::*;
+
+use crate::constants::*;
+use crate::errors::*;
+use crate::state::*;
+
+#[derive(Accounts)]
+pub struct VerifyProfileForCommunity<'info> {
+    #[account(mut)]
+    pub community: Account<'info, Community>,
+    #[account(mut)]
+    pub profile: Account<'info, Profile>,
+    #[account(
+        init,
+        payer = fee_payer,
+        space = CommunityVerification::size(),
+        seeds = [
+            PREFIX,
+            COMMUNITY_VERIFICATION,
+            community.key().as_ref(),
+            profile.key().as_ref(),
+        ],
+        bump
+    )]
+    pub community_verification: Account<'info, CommunityVerification>,
+    #[account(constraint = authority.key() == community.authority @ PubkeyProfileError::UnauthorizedCommunityAction)]
+    pub authority: Signer<'info>,
+    #[account(mut)]
+    pub fee_payer: Signer<'info>,
+    pub system_program: Program<'info, System>,
+}
+
+pub fn verify_profile(ctx: Context<VerifyProfileForCommunity>) -> Result<()> {
+    let community = &ctx.accounts.community;
+    let profile = &ctx.accounts.profile;
+    let community_verification = &mut ctx.accounts.community_verification;
+    let authority = &ctx.accounts.authority;
+
+    // Set up the CommunityVerification account
+    community_verification.set_inner(CommunityVerification {
```

Comment:
Let's go over the spec again before moving on with this. We don't need a separate account for reach verification. Instead, as per the spec, we tack it into the `communities` array on the identity being verified.

Reply:
This has been fixed. Both pointers exist in the Profile and Community Account structs respectively now